client/src/components/ProfilePicture/NewUserUploadPicture.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import './NewUserUploadPicture.css'
+import Button from '../Button/Button'
+
+const handleClick = e => {
+  console.log("Clicked");
+}; //from ../Button/Button.stories.js
+
+const NewUserUpload = () => {
```

Comment:
`NewUser` seems like contextual information - there may be places where we want the user to upload something that has nothing to do with creating a new user profile. Maybe we could call this `FileUpload`, or something like that?